drivers/media/platform/bcm2835/bcm2835-unicam.c

```diff
@@ -2293,30 +2354,6 @@ static void unicam_return_buffers(struct unicam_node *node,
 	spin_unlock_irqrestore(&node->dma_queue_lock, flags);
 }
 
-static int unicam_check_format(struct unicam_node *node)
```

Comment:
I think this hunk should be squashed with the previous commit - it was added there and not used.